pharo-repository/AWS-Core/AWSConfig.class.st

```diff
@@ -85,12 +85,13 @@ AWSConfig >> hostUrl: anObject [
 
 { #category : #accessing }
 AWSConfig >> regionName [
-	^self at: #regionName ifAbsentPut: [ self defaultRegionName ]
+	^ self at: #regionName ifAbsentPut: [ self defaultRegionName ]
 ]
 
 { #category : #accessing }
 AWSConfig >> regionName: anObject [
-	^self at: #regionName put: anObject
+	settingsDict removeKey: #key ifAbsent: [  ].
```

Comment:
Why is this fix needed?

Reply:
Damn it, I messed up with Iceberg... committing wrong stuff. 
my idea was to clear the hostUrl setting when we update the region because the hosturl is lazily initialize using the region setting... so once you changed region, the hosturl is wrong, and you must reset your settings manually. 

Anyway I will update the PR, with much clearer stuff and some tests.